### PR TITLE
Fixed some issues in the samples

### PR DESCRIPTION
**Pull request by jproust**
All samples now build from root folder

### CODE REVIEW

File: src/samples/visualiser/pygletExample.py [hunk 1]
```diff
@@ -314,7 +314,7 @@ def __init__(self, width, height, x, y, target):
     parent = target.parent
     super().__init__(width, height, x, y, parent)
     self.camera = MapCamera(self.renderTarget, target.camera, 0.3)
-    self.skyboxTexture = pyglet.image.load("parchment.jpg").get_texture()
+    self.skyboxTexture = pyglet.image.load("src\\samples\\visualiser\\parchment.jpg").get_texture()
```
Comment (bwockner) [inline src/samples/visualiser/pygletExample.py:317]:
```suggestion
    self.skyboxTexture = pyglet.image.load("src/samples/visualiser/parchment.jpg").get_texture()
```
Unsure if the alternative breaks linux compatibility

File: src/samples/visualiser/pygletExample.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ class UDViewPort():
   """This class represents the quad that the UDS render is blitted to,
   it handles the camera information associated with the view it controls
   """
-  def __init__(self, width, height, x, y, parent, skyboxImage="WaterClouds.jpg"):
+  def __init__(self, width, height, x, y, parent, skyboxImage="src\\samples\\visualiser\\WaterClouds.jpg"):
```
Comment (bwockner) [inline src/samples/visualiser/pygletExample.py:33]:
```suggestion
  def __init__(self, width, height, x, y, parent, skyboxImage="src/samples/visualiser/WaterClouds.jpg"):
```

File: src/samples/basic_convert.py [hunk 3]
```diff
@@ -16,11 +17,12 @@
   convertContext = udSDKConvert.udConvertContext(context)
 
   #add the input files:
-  convertContext.add_item("../samplefiles/sampleInput.las")
+  convertContext.add_item(abspath("./samplefiles/sampleInput.las"))
+  # convertContext.add_item("../../samplefiles/sampleInput.las")
```
Comment (bwockner) [inline src/samples/basic_convert.py:21]:
```suggestion

```

**Review by bwockner — CHANGES_REQUESTED** [review]

**Review by bwockner — APPROVED** [review]

File: src/udSDK/udSDKProject.py [hunk 4]
```diff
@@ -252,7 +252,7 @@ def nextSibling(self):
         The next node at the same level in the tree, None if this is the last node in this level
         """
         if self.parent is not None:
-            return self.parent.child_from_pointer(self.pNextSibling)
+            return self.from_pointer(self.pNextSibling)
         else:
             return None
```
Comment (samsinsane) [inline src/udSDK/udSDKProject.py:257]: This doesn't need to do the check for `self.parent` anymore, this should work the same as `firstChild`.

Comment (bwockner) [reply src/udSDK/udSDKProject.py:257]: Worse- due to a botched renaming child_from_pointer doesn't exist any more 
I'll address this in a separate PR

Comment (samsinsane) [reply src/udSDK/udSDKProject.py:257]: It's not using `child_from_pointer`, it's using `from_pointer`?

Comment (bwockner) [reply src/udSDK/udSDKProject.py:257]: My bad- misread what was going on - fixed in #48

**Pull request merged**